specification/web/resource-manager/Microsoft.Web/stable/2023-12-01/CommonDefinitions.json

```diff
@@ -952,6 +952,93 @@
         }
       }
     },
+    "SwiftVirtualNetwork": {
+      "description": "Swift Virtual Network Contract",
+      "type": "object",
+      "properties": {
+        "subnetResourceId": {
+          "description": "The Virtual Network subnet's ARM ID",
+          "type": "string"
+        },
+        "swiftSupported": {
```

Comment:
>swiftSupported [](http://example.com/codeflow?start=9&length=14)

Recommend enum instead of boolean, even if there are only two values. Something like: integration: { Default, Swift } or similar.

Reply:
This property is in use and has been for long. Swagger Definition was just not defined until now, so we cannot make breaking changes.